.devcontainer/Dockerfile

```diff
@@ -217,49 +217,47 @@ RUN apt update && apt upgrade -y --autoremove \
         time \
         unzip \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
-
-RUN sed -i 's/^# *\(en_US.UTF-8\)/\1/' /etc/locale.gen && locale-gen
-RUN (echo "LC_ALL=en_US.UTF-8" \
-    && echo "LANGUAGE=en_US.UTF-8") >/etc/default/locale
+RUN sed -i 's/^# *\(en_US.UTF-8\)/\1/' /etc/locale.gen \
+    && locale-gen \
+    && (echo "LC_ALL=en_US.UTF-8" && echo "LANGUAGE=en_US.UTF-8") >/etc/default/locale
+RUN groupadd --gid=1000 code \
+    && useradd --create-home --uid=1000 --gid=1000 code \
+    && echo "code ALL=(root) NOPASSWD:ALL" >/etc/sudoers.d/code \
+    && chmod 0440 /etc/sudoers.d/code
 
 # Copy node utilities first, because the bin directory include symlinks that we want to preserve.
-COPY --from=node /usr/local/lib/node_modules /usr/local/lib/node_modules
-COPY --from=node /usr/local/include/node /usr/local/include/node
 COPY --from=node /usr/local/man/man1/node.1 /usr/local/man/man1/
+COPY --from=node /usr/local/include/node /usr/local/include/node
+COPY --from=node /usr/local/lib/node_modules /usr/local/lib/node_modules
```

Comment:
is there a reason these were rearranged? not a big deal.

Reply:
eh, this is mostly an artifact of changing perms on this (and then undoing it).